static/js/containers/ProfilePage_test.js

```diff
@@ -114,16 +114,15 @@ describe("ProfilePage", function() {
 
   describe('switch toggling behavior', () => {
     beforeEach(() => {
-      let userProfile = Object.assign({}, _.cloneDeep(USER_PROFILE_RESPONSE), {
+      let userProfile = {
+        ...USER_PROFILE_RESPONSE,
         education: [],
         work_history: []
-      });
+      };
       helper.profileGetStub.
         withArgs(SETTINGS.user.username).
         returns(
-          Promise.resolve(Object.assign({}, userProfile, {
-            username: SETTINGS.user.username
-          }))
+          Promise.resolve(userProfile)
```

Comment:
maybe move this argument up to the line above? `returns(Promise.resolve(userProfile))`